kubemarine/procedures/check_iaas.py

```diff
@@ -242,11 +242,12 @@ def system_distributive(cluster):
         detected_supported_os = []
         detected_unsupported_version = []
         supported_versions = []
+        detected_versions_compatability = []
```

Comment:
It is simpler to use `set()`. In this case it is enough to add to the set and finally check that len() == 1.